common/tests/call_once_int/call_once_int.c

```diff
@@ -2,17 +2,22 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 #ifdef __cplusplus
+#include <cstdint>
 #include <cstdlib>
 #include <ctime>
 #else
+#include <stdbool.h>
+#include <stdint.h>

```

Comment:
I am confused. Isn't this also building with iwyu on Win?

Reply:
Jelani asked the same about my umock changes. Not sure why iwyu misses some of these with Clang+cl.exe but catches them for Linux Clang 6.0...

---
In reply to: [564146942](https://github.com/Azure/c-pal/pull/69#discussion_r564146942) [](ancestors = 564146942)